apps/aecore/src/aec_peers.erl

```diff
@@ -51,14 +54,14 @@
 %%------------------------------------------------------------------------------
 %% Add peer by url or supplying full peer() record. Try to connect.
 %%------------------------------------------------------------------------------
--spec add(uri() | peer()) -> ok.
+-spec add(http_uri:uri() | peer()) -> ok.
```

Comment:
If touching the spec-ing using this type, I understand this is not the way to go as this type is documented but not actually exported by `http_uri` module.